.github/workflows/release.yml

```diff
@@ -40,9 +46,15 @@ jobs:
     name: Create release on self-hosted dogfood instance
     needs: release
     steps:
+      - name: Get auth token
+        id: token
+        uses: actions/create-github-app-token@5d869da34e18e7287c1daad50e0b8ea0f506ce69 # v1.11.0
+        with:
+          app-id: ${{ vars.SENTRY_RELEASE_BOT_CLIENT_ID }}
+          private-key: ${{ secrets.SENTRY_RELEASE_BOT_PRIVATE_KEY }}
       - uses: actions/checkout@v4
         with:
-          token: ${{ secrets.GH_RELEASE_PAT }}
+          token: ${{ steps.token.outputs.token }}
```

Comment:
I don't think we need a special token here. @hubertdeng123 @chadwhitacre can you weigh in? Why is `GITHUB_TOKEN` not enough for the release?

Reply:
Hmm I would think we shouldn't need this either, this should just be for a release on our self-hosted dogfood instance so it would surprise me if we need this here. I think we can omit the token completely for the checkout step here

Reply:
Thanks for the insight! Removed it and let's see how it goes, we can always add it back if figured out later that it's actually needed.